CHANGELOG.md

```diff
@@ -16,6 +16,53 @@ Mapbox welcomes participation and contributions from everyone.
 ## Dependencies
 * Update mapbox-gestures-android dependency to [v0.8.0](https://github.com/mapbox/mapbox-gestures-android/releases/tag/v0.8.0). ([1645] (https://github.com/mapbox/mapbox-maps-android/pull/1645))
 
+# 10.8.0 September 7, 2022
+## Features ✨ and improvements 🏁
+* Introduce a callback to be invoked when the device compass sensors need to be re-calibrated. ([1513](https://github.com/mapbox/mapbox-maps-android/pull/1513))
+* Add support for `LocationComponentSettingsInterface.pulsingMaxRadius` to follow location's accuracy radius. ([1561](https://github.com/mapbox/mapbox-maps-android/pull/1561))
+* Avoid map content disappearing on the sides of the screen when LOD is enabled ([1650](https://github.com/mapbox/mapbox-maps-android/pull/1650))
+* Enable [pitch] and [distance-from-center] style expressions ([1650](https://github.com/mapbox/mapbox-maps-android/pull/1650))
+* Enable ambient occlusion for 3D buildings and provide paint property in model layer to control it. ([1650](https://github.com/mapbox/mapbox-maps-android/pull/1650))
+* Add custom layer support for rendering (draping) to terrain and globe surface. (https://github.com/mapbox/mapbox-gl-js/pull/11996) ([1650](https://github.com/mapbox/mapbox-maps-android/pull/1650))
```

Comment:
pls also remove these entries

Reply:
The features & improvements section should be the same as the entries in 10.8.0-beta.1